pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -60,7 +60,6 @@ func CreateK8sBindingInjector() (BindingInjector, error) {
 
 func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *brokerapi.Credential) error {
 	is := makeInjectionSet(binding, cred)
-
 	if err := b.injectSecret(is.secret); err != nil {
```

Comment:
I :100: agree. For context, I held off on making refactors in this PR, but may as well clean this file up.

Changes incoming